### PR TITLE
S k song Update native docs part II: native/guides/ui/efl and dali

### PR DESCRIPTION
**Pull request by journey2w**
- Merge branch 'Update-native-docs-part-II' of https://github.com/s-k-song/tizen-docs into s-k-song-Update-native-docs-part-II
- Separate directories of components for Mobile and Wearable

### CODE REVIEW

File: docs/application/native/guides/ui/dali/touchdata.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ DALi informs you with a callback when the user touches the screen and when the t
 
 The `Dali::TouchData` class replaces the old deprecated `Dali::TouchEvent` struct. It contains functions for retrieving the information that the `Dali::TouchEvent` struct provided, but also adds the radius, touch pressure, and angle information.
 
-​        **Note**  
-        > Do not use the `Dali::TouchData` class in a container.	 As the `Dali::TouchData` class is a handle to an internal object, it must not be copied (or used in a container) as all that happens is that the handle is copied to the same object, and the internal object can change unexpectedly. If the data must be stored in the application, save only the required data (retrieved using the class functions).    
+​> **Note**
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/dali/touchdata.md]: It looks like it need space. The note style is not applied.

File: docs/application/native/guides/ui/efl/component-custom.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ To customize a UI component:
 
 1. Create a project for a basic EDC UI application.
 
-   See [Creating Your First Tizen Mobile Native Application](../../../getting-started/mobile/first-app.md) or [Creating Your First Tizen Wearable Native Application](../../../getting-started/wearable/first-app.md) for creating a EDC UI project.
+   See [Creating Your First Tizen Mobile Native Application](../../../../getting-started/mobile/first-app.md) or [Creating Your First Tizen Wearable Native Application](../../../../getting-started/wearable/first-app.md) for creating a EDC UI project.
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/component-custom.md]: Broken links : I think we should remove one `../` from each links.

File: docs/application/native/guides/ui/dali/touchdata.md (hunk 1 above)
Comment (journey2w) [reply docs/application/native/guides/ui/dali/touchdata.md]: done

File: docs/application/native/guides/ui/efl/mobile/ui-components.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Mobile UI Components
+
+EFL Elementary is a set of fast, finger-friendly, scalable, and themeable UI component libraries. The number of supported UI components is around 80, including both [containers](./ui-layouts.md) and non-containers (listed in this topic). Originally, Elementary was developed as part of the Window Manager development on desktop devices. For the mobile profile, Tizen reused the proper UI components and created new ones, and then enhanced and adjusted all of them for Tizen native applications.
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/mobile/ui-components.md]: Broken link. `./ui-layouts.md` should be changed into `../ui-layouts.md`.

File: docs/application/native/guides/ui/efl/wearable/component-circle-datetime.md [hunk 4]
```diff
@@ -56,7 +56,7 @@ To configure the circle properties of the circle datetime:
 
   To change the item properties, use the `eext_circle_object_item_XXX()` functions.
 
-For more information, see [Circle Object](component-circ-object-w.md) and the [Efl Extension Circle Object](../../../../../org.tizen.native.wearable.apireference/group__CAPI__EFL__EXTENSION__CIRCLE__OBJECT__MODULE.html) API.
+For more information, see [Circle Object](component-circ-object.md) and the [Efl Extension Circle Object](../../../../../org.tizen.native.wearable.apireference/group__CAPI__EFL__EXTENSION__CIRCLE__OBJECT__MODULE.html) API.
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/wearable/component-circle-datetime.md]: Broken link : `component-circ-object.md` should be changed into `component-circle-object.md`.

File: docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md [hunk 5]
```diff
@@ -80,8 +80,8 @@ To add an item:
 
    | Text                 | Setting function                         | View                                     |
    |--------------------|----------------------------------------|----------------------------------------|
-   | `selector,main_text` | `eext_rotary_selector_item_part_text_set()``eext_rotary_selector_item_domain_translatable_part_text_set()` | ![main_text](./media/rotary_selector_main_text.png) |
-   | `selector,sub_text`  | `eext_rotary_selector_item_part_text_set()``eext_rotary_selector_item_domain_translatable_part_text_set()` | ![sub_text](./media/rotary_selector_sub_text.png) |
+   | `selector,main_text` | `eext_rotary_selector_item_part_text_set()`<br> `eext_rotary_selector_item_domain_translatable_part_text_set()` | ![main_text](./media/rotary_selector_main_text.png) |
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md]: Broken images in line 83 and lin3 84. It looks like images are in wrong folder. They are in `../media/` not in `./media/`.

File: docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md [hunk 6]
```diff
@@ -95,8 +95,8 @@ The following table shows the configurable rotary selector parts.
 |----------------------------------------|----------------------------------------|----------------------------------------|
 | `selector,main_text`                     | `eext_rotary_selector_item_part_color_set()` | ![main_text](./media/rotary_selector_main.png) |
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md]: Broken images in this chart. It looks like images are in wrong folder. They are in `../media/` not in `./media/`.

File: docs/application/native/guides/ui/efl/wearable/component-progressbar.md [hunk 7]
```diff
@@ -124,14 +124,14 @@ The following table lists the available component styles according to the UI scr
 
 **Table: Progressbar styles**
 
-| Style                                    | Sample                                   |                                          |
+| Style                                    | Sample                                   |
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/wearable/component-progressbar.md]: Table is broken.

File: docs/application/native/guides/ui/efl/mobile/ui-components.md (hunk 3 above)
Comment (journey2w) [reply docs/application/native/guides/ui/efl/mobile/ui-components.md]: done

File: docs/application/native/guides/ui/efl/wearable/component-circle-datetime.md (hunk 4 above)
Comment (journey2w) [reply docs/application/native/guides/ui/efl/wearable/component-circle-datetime.md]: done

File: docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md (hunk 5 above)
Comment (journey2w) [reply docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md]: done

File: docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md (hunk 6 above)
Comment (journey2w) [reply docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md]: done

File: docs/application/native/guides/ui/efl/wearable/component-progressbar.md (hunk 7 above)
Comment (journey2w) [reply docs/application/native/guides/ui/efl/wearable/component-progressbar.md]: done.

**Pull request merged**